packages/inference/package.json

```diff
@@ -52,6 +52,7 @@
 		"check": "tsc"
 	},
 	"dependencies": {
+		"@huggingface/hub": "workspace:^",
```

Comment:
Just checking, this add dep is fine @SBrandeis? (also @coyotte508)

Reply:
Should edit https://github.com/huggingface/huggingface.js/blob/261eba8845f14623db7a11f03f6caff1430ba736/.github/workflows/inference-publish.yml#L57 to also check hub then

Reply:
i see, then it's maybe simpler to use the same pattern as in https://github.com/huggingface/huggingface.js/blob/905c8cd451be669462767b0e9ce0c367e2b15216/packages/inference/src/lib/getDefaultTask.ts#L41 i.e. make an ad hoc call (not using the hub package)

Reply:
sure